packages/ui/src/services/listCourtCases.ts

```diff
@@ -16,6 +16,9 @@ import leftJoinAndSelectTriggersQuery from "./queries/leftJoinAndSelectTriggersQ
 import QueryColumns from "./QueryColumns"
 import { formatName } from "../helpers/formatName"
 
+const getExcludedTriggers = (excludedTriggers?: string[]): string[] =>
+  excludedTriggers && excludedTriggers.length > 0 ? excludedTriggers : [""]
```

Comment:
I had tried it but then it may return undefined if excludedTriggers is undefined, whereas we strictly want to return an array (either with excluded triggers or empty)